hbjs.js

```diff
@@ -533,7 +533,23 @@ function hbjs(Module) {
     return trace;
   }
 
-  function get_version() {
+  function version() {
+    var major = exports.malloc(4);
```

Comment:
In theory you can do with one malloc?

Reply:
How is that?

Reply:
I was thinking allocing 12 bytes and sending in three chunks of it somehow... Nevermind though.

Reply:
I see. I don’t think it is a real malloc though, so probably it does not make much difference.

Reply:
Something like this I guess:
```js
    var versionPtr = exports.malloc(4 * 3);
    var major = versionPtr;
    var minor = versionPtr + 4;
    var micro = versionPtr + 8;
```

Reply:
https://github.com/harfbuzz/harfbuzzjs/commit/fb92648599d11635b656715b158d538a6cbbae7f